Resources/Prototypes/_Sunrise/Catalog/uplink_catalog.yml

```
@@ -381,16 +381,56 @@
       - NukeOpsUplink
 
 - type: listing
-  id: UplinkMagazineBauer127Extended
-  name: uplink-magazine-bauer127-extended-name
-  description: uplink-magazine-bauer127-extended-desc
+  id: UplinkMagazineBauer127
+  name: uplink-magazine-bauer127-name
+  description: uplink-magazine-bauer127-desc
   icon: { sprite: _Sunrise/Objects/Weapons/Guns/Ammunition/Magazines/Rifle/sr127_mag.rsi, state: base }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)